test/e2e/cmd/cmd.go

```diff
@@ -69,13 +69,15 @@ func New() *cobra.Command {
 
 func newDeployCmd(def *app.Definition) *cobra.Command {
 	var cfg app.DeployConfig
+
 	pingPongN := defaultPingPongDeploy // Default to 1000 ping pongs.
+	cfg.PingPongN = &pingPongN
```

Comment:
so now you'll need to align this with our general config/flag pattern:
- default default flag values in `DefaultDeployConfig` function (in app pkg next to type def).
- Define flags of config struct in one place. So move `--ping-poing` below into `bindDeployFlags`

Reply:
Awesome! I'll work on this "tomorrow" wanted to ask what was the right way on how to do it was but I didn't want to create noise too late at night for everyone

Reply:
@corverroos done, if you could check to confirm it that would be great

Reply:
Cool, looks good I added ` app.DefaultDeployConfig` function to align with our flags pattern. Also only bound `PromSecrets` during E2E fags.